Front/client/components/Intro.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+class Intro extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { // начали с конструктора и примерно прикинули что зайдет в стейт
```

Comment:
Application UI state should be placed in top level component and then be passing down. In your case, App or Main component should contain state.